Lab/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Lab
+{
+     public class Program
+    {
+        public static double balance = 0;
+        public static void Main(string[] args)
+        {
+            Console.WriteLine(ViewBalance(balance));
+        }
+
+        public static double ViewBalance(double balance)
+        {
+            return balance;
+        }
+
+        /// <summary>
+        /// Withdraws money from ATM. Sets balance to a lower amount
+        /// </summary>
+        /// <param name="amount">amount to be withdrawn</param>
```

Comment:
Good place to document limitations, i.e. greater than zero and less than or equal to current balance.